packages/xo-web/src/xo-app/proxies/index.js

```diff
@@ -109,7 +109,7 @@ const COLUMNS = [
         data-name='name'
         data-proxy={proxy}
         onChange={_editProxy}
-        value={defined(proxy.name, '')}
+        value={proxy.name}
```

Comment:
What if the name has been deleted after the creation?

Reply:
The API don't provide a way to delete it.

Reply:
Not necessarily directly from the API but an internal workflow could delete it so we might want to handle it anyway, don't you think?

Reply:
IMHO, if it's an internal code which delete it, it'll be a bug and we'll fix it when it'll produce.